### DISCUSSION TITLE
not giving random user

### DISCUSSION THREAD

**Discussion by kantoash** [faker-js/faker#1119] [Q&A]
error `Module not found: Can't resolve 'faker'
  1 | import React from 'react'
> 2 | import  faker  from 'faker'
  3 | import { useEffect } from 'react'
  4 | 
  5 | function Stories() {

Import trace for requested module:
./components/Feed.js
./pages/index.tsx`
code : here
`import React from 'react'
import  faker  from 'faker'
import { useEffect } from 'react'

function Stories() {

   useEffect(() => {
    const suggestions = [...Array(20)].map((_, i) => ({
        ...faker.helpers.contextualCard(),
        id: i,
    }));
    console.log(suggestions);
   }, [])
   
    
  return (
    <div>

    </div>
  )
}
`

**Comment by prisis** [comment]
Hey, please read the documentation and the readme of the project it will fix your issue.
